docker-compose-ci.yaml

```diff
@@ -0,0 +1,24 @@
+version: "3.3"
+services:
+  postgres:
+    image: postgres:9.4
+    logging:
+      driver: none
+    environment: 
+      POSTGRES_HOST_AUTH_METHOD: trust
+    volumes: 
+      - ./psql-users.sh:/docker-entrypoint-initdb.d/psql-users.sh
+
+  app:
+    image: quay.io/ncigdc/jenkins-agent:multipython
+    environment:
+      SSH_AUTH_SOCK: $SSH_AUTH_SOCK
+      HTTPS_PROXY: http://cloud-proxy:3128
+      HTTP_PROXY: http://cloud-proxy:3128
+    volumes:
+    - .:/home/jenkins
+    - $SSH_AUTH_SOCK:$SSH_AUTH_SOCK
+    command: bash -c "mkdir -p ~/.ssh && echo -e 'Host github.com\n\tStrictHostKeyChecking no\n' >> ~/.ssh/config && rm -rf .tox && find . \( -name '__pycache__' -or -name '*.pyc' \) -delete  && ./wait-for-it.sh localhost:5432 -t 120 &&  pip install tox -q --user && tox"
```

Comment:
I think you should be able to do similar thing with `tox -r` like you did in other repos?

Reply:
👍 I have updated to only include what is _really_ needed.